mirdata/datasets/irmas.py

```diff
@@ -221,19 +221,32 @@ def __init__(self, track_id, data_home):
 
     @core.cached_property
     def instrument(self):
-        """(list, string): predominant instrument"""
+        """Predominant instrument
+
+        Returns:
+            (list, string): Predominant instrument
```

Comment:
We have this in two formats: (list, string) and `list of strings`, which one should we keep @rabitt?

Reply:
@PRamoneda maybe you know what's better? Is the only thing left and it will be ready to merge IMO

Reply:
I think this option is not right. This more or less 'list of strings'. I think the correct one is list[str] jajajjaa. Because if not code lint scripts don't work. But well maybe is best to ask @rabitt.

Reply:
@PRamoneda is right - `(list, string)` would mean it returns a tuple, which has first a list and then a string

I think the "right" way is (list): Predominant instruments (as str)

Reply:
I'll update this last change and merge :)